frontend/src/app/main/ui/workspace/sidebar/options/shapes/multiple.cljs

```diff
@@ -212,17 +213,24 @@
             :else                  (attrs/get-attrs-multi [v1 v2] attrs)))
 
         merge-token-values
-        (fn [acc keys attrs]
-          (reduce
-           (fn [accum key]
-             (let [new-val (get attrs key)
-                   existing (get accum key ::not-found)]
-               (cond
-                 (= existing ::not-found) (assoc accum key new-val)
-                 (= existing new-val)     accum
-                 :else                    (assoc accum key :multiple))))
-           acc
-           keys))
+        (fn [acc shape-attrs applied-tokens]
```

Comment:
A nested reduce makes this function really hard to follow and is not a good practice. 
Could we simplify it somehow?